website/project/taxonomies/__init__.py

```diff
@@ -13,10 +13,22 @@ class Subject(StoredObject):
     _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
     text = fields.StringField(required=True)
     parents = fields.ForeignField('subject', list=True)
+    children = fields.ForeignField('subject', list=True)
 
     @property
     def absolute_api_v2_url(self):
         return api_v2_url('taxonomies/{}/'.format(self._id))
 
+    @property
+    def child_count(self):
+        return len(self.children)
+
+    def set_children(self):
```

Comment:
This seems like something that should go into the update script, not on the model, given that it's a noop after being called once.